server/plugin/plugin.go

```diff
@@ -92,11 +93,30 @@ func (p *Plugin) OnActivate() error {
 		return errors.Wrap(err, "failed to register command")
 	}
 
+	// Telemetry client
 	p.telemetryClient, err = telemetry.NewRudderClient()
 	if err != nil {
-		p.env.bot.Errorf("Cannot create telemetry client. err=%v", err)
+		p.API.LogWarn("Telemetry client not started", "error", err.Error())
 	}
 
+	// Get config values
+	p.updateEnv(func(e *Env) {
+		e.Dependencies.Tracker = tracker.New(
+			telemetry.NewTracker(
+				p.telemetryClient,
+				p.API.GetDiagnosticId(),
+				p.API.GetServerVersion(),
+				e.PluginID,
+				e.PluginVersion,
+				config.TelemetryShortName,
+				telemetry.NewTrackerConfig(p.API.GetConfig()),
+				logger.New(p.API),
+			),
+		)
+		e.bot = e.bot.WithConfig(stored.Config)
+		e.Dependencies.Store = store.NewPluginStore(p.API, e.bot, e.Dependencies.Tracker)
```

Comment:
I've had to add this to prevent a panic from happening.

Sometimes the hooks are called: onConfChange + Onactivate (without a second onConfChange), and that ended up with nil trackers stored across subpackages like flow.